lib/Runtime/Library/SparseArraySegment.inl

```diff
@@ -241,7 +241,7 @@ namespace Js
             }
             for (i = (start + step-1)/step; i < (size/step); i++)
             {
-                ((Var*)(this->elements))[i] = fill;
+                ((Var*)(this->elements))[i] = fill; // swb: no write barrier, set to non-GC pointer
```

Comment:
do we need to add a global NoWriteBarrierSet to do this? the casting looks not straightforward.